bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF

```diff
@@ -18,4 +18,6 @@ Import-Package: groovy.lang,
  org.junit,
  org.junit.runner,
  org.junit.runners,
- org.osgi.service.cm
+ org.osgi.service.cm,
+ org.mockito
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
The only thing to do is to add "org.mockito.stubbing" to your package imports:

```diff
diff --git a/bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF b/bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF
index 889feb68c..b83e1b376 100644
--- a/bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF
+++ b/bundles/core/org.eclipse.smarthome.core.test/META-INF/MANIFEST.MF
@@ -19,5 +19,5 @@ Import-Package: groovy.lang,
  org.junit.runner,
  org.junit.runners,
  org.osgi.service.cm,
- org.mockito
-Require-Bundle: org.junit,org.mockito,org.hamcrest
+ org.mockito,
+ org.mockito.stubbing
```

Or am I missing something. Can you give it a try?